### ISSUE TITLE
Example in main README.md does not work

### ISSUE THREAD

**Issue by maciejjaskowski** [tensorflow/tensor2tensor#95]
It fails on `t2t-datagen` with different errors for both v1.0.10 and v1.0.9. v1.0.8 seems to work.

v1.0.10 (from source code)
```
INFO:tensorflow:Trying min_count 500
INFO:tensorflow:Iteration 0
Traceback (most recent call last):
  File "/home/jaskowski/.virtualenvs/gripper1/bin/t2t-datagen", line 378, in <module>
    tf.app.run()
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensorflow/python/platform/app.py", line 48, in run
    _sys.exit(main(_sys.argv[:1] + flags_passthrough))
  File "/home/jaskowski/.virtualenvs/gripper1/bin/t2t-datagen", line 361, in main
    training_gen(), FLAGS.problem + UNSHUFFLED_SUFFIX + "-train",
  File "/home/jaskowski/.virtualenvs/gripper1/bin/t2t-datagen", line 151, in <lambda>
    lambda: wmt.ende_wordpiece_token_generator(FLAGS.tmp_dir, True, 2**15),
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/wmt.py", line 230, in ende_wordpiece_token_generator
    tmp_dir, "tokens.vocab.%d" % vocab_size, vocab_size)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/generator_utils.py", line 265, in get_or_generate_vocab
    vocab_size, tokenizer.token_counts, 1, 1e3)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 343, in build_to_target_size
    return bisect(min_val, max_val)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 329, in bisect
    present_count, num_iterations)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 377, in build_from_token_counts
    escaped_token = self._escape_token(token)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 476, in _escape_token
    ret += c
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 0: ordinal not in range(128)
```

v1.0.9 (from pip)
```INFO:tensorflow:Iteration 3
INFO:tensorflow:vocab_size = 34813
[109, 17123, 62, 29963, 22, 41, 4, 19147, 12493, 10749, 34621, 24836, 20, 3]
[u'This_', u'sentence_', u'was_', u'encode', u'd_', u'by_', u'the_', u'Sub', u'word', u'Tex', u't', u'Enco', u'der_', u'._']
This sentence was encoded by the SubwordTextEncoder.
Traceback (most recent call last):
  File "/home/jaskowski/.virtualenvs/gripper1/bin/t2t-datagen", line 378, in <module>
    tf.app.run()
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensorflow/python/platform/app.py", line 48, in run
    _sys.exit(main(_sys.argv[:1] + flags_passthrough))
  File "/home/jaskowski/.virtualenvs/gripper1/bin/t2t-datagen", line 361, in main
    training_gen(), FLAGS.problem + UNSHUFFLED_SUFFIX + "-train",
  File "/home/jaskowski/.virtualenvs/gripper1/bin/t2t-datagen", line 151, in <lambda>
    lambda: wmt.ende_wordpiece_token_generator(FLAGS.tmp_dir, True, 2**15),
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/wmt.py", line 230, in ende_wordpiece_token_generator
    tmp_dir, "tokens.vocab.%d" % vocab_size, vocab_size)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/generator_utils.py", line 265, in get_or_generate_vocab
    vocab_size, tokenizer.token_counts, 1, 1e3)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 329, in build_to_target_size
    return bisect(min_val, max_val)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 323, in bisect
    other_subtokenizer = bisect(min_val, present_count - 1)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 323, in bisect
    other_subtokenizer = bisect(min_val, present_count - 1)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 323, in bisect
    other_subtokenizer = bisect(min_val, present_count - 1)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 323, in bisect
    other_subtokenizer = bisect(min_val, present_count - 1)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 323, in bisect
    other_subtokenizer = bisect(min_val, present_count - 1)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 323, in bisect
    other_subtokenizer = bisect(min_val, present_count - 1)
  File "/home/jaskowski/.virtualenvs/gripper1/local/lib/python2.7/site-packages/tensor2tensor/data_generators/text_encoder.py", line 324, in bisect
    if (abs(other_subtokenizer.vocab_size - target_size) <
AttributeError: 'NoneType' object has no attribute 'vocab_size'```

**Comment by stefan-it** [comment]
It's known problem, see more information [here](https://github.com/tensorflow/tensor2tensor/issues/74#issuecomment-312757562) :)

**Comment by lukaszkaiser** [comment]
This is hopefully corrected in 1.0.11. please give it a try. I'm closing for now, please reopen if you still see the issue.
